roles/ec2-app/tasks/app-asg.yml

```diff
@@ -74,7 +74,7 @@
         name: "{{ asg_name }}-{{ opg_data.stack }}"
         launch_config_name: "{{ asg_name }}-{{ opg_data.stack }}-{{ lc_pf.stdout }}"
         health_check_period: 900
-        health_check_type: 'ELB'
+        health_check_type: "{{ appdata.asg.health_check|default('ELB') }}"
```

Comment:
this answered my own question:
https://gitlab.service.opg.digital/opsforks/opg-core-deploy/merge_requests/66/diffs